sdcm/mgmt/operator.py

```diff
@@ -0,0 +1,346 @@
+        return {mapping[key]: value for key, value in data.items()
+                if value is not None and mapping.get(key, None) is not None}
+
+    def to_dict(self, remove_defaults=True) -> dict:
+        result = asdict(self)
+        if remove_defaults:
+            return {key: val for key, val in result.items() if val is not None}
+        return result
+
+    @classmethod
+    def from_dict(cls, data: dict, **kwargs):
+        field_names = [field.name for field in fields(cls)]
+        return cls(**{key: val for key, val in data.items() if key in field_names}, **kwargs)
+
+
+@dataclass
+class ScyllaOperatorTaskBaseClass(BaseClass):
+    name: 'str'
+    # StartDate specifies the task start date expressed in the RFC3339 format or now[+duration],
+    # e.g. now+3d2h10m, valid units are d, h, m, s (default "now").
+    startDate: str = None
+    # Interval task schedule interval e.g. 3d2h10m, valid units are d, h, m, s (default "0").
+    interval: str = None
+    # NumRetries the number of times a scheduled task will retry to run before failing (default 3).
+    numRetries: int = None
+
+
+@dataclass
+class ScyllaOperatorRepairTask(ScyllaOperatorTaskBaseClass):
+    # DC list of datacenter glob patterns, e.g. 'dc1', '!otherdc*' used to specify the DCs
+    # to include or exclude from backup.
+    dc: List[str] = None
+    # FailFast stop repair on first error.
+    failFast: bool = None
+    # Intensity integer >= 1 or a decimal between (0,1), higher values may result in higher speed and cluster load.
+    # 0 value means repair at maximum intensity.
+    intensity: float = None
+    # Parallel The maximum number of repair jobs to run in parallel, each node can participate in at most one repair
+    # at any given time. Default is means system will repair at maximum parallelism.
+    parallel: int = None
+    # Keyspace a list of keyspace/tables glob patterns, e.g. 'keyspace,!keyspace.table_prefix_*'
+    # used to include or exclude keyspaces from repair.
+    keyspace: List[str] = None
+    # SmallTableThreshold enable small table optimization for tables of size lower than given threshold.
+    # Supported units [B, MiB, GiB, TiB] (default "1GiB").
+    smallTableThreshold: str = None
+
+
+@dataclass
+class ScyllaOperatorRepairTaskStatus(ScyllaOperatorRepairTask):
+    # These statuses are not available at scylla-operator 1.0
+    # You have to relay on task status in manager
+
+    STARTED = 'STARTED'
+    COMPLETED = 'COMPLETED'
+    SCHEDULED = 'SCHEDULED'
+    ERROR = 'ERROR'
+    IN_PROGRESS = 'IN PROGRESS'
+
+    error: str = None
+    status: str = None
+    id: str = None
+
+    @property
+    def mgmt_task_id(self) -> str:
+        return f'repair/{self.id}'
+
+
+@dataclass
+class ScyllaOperatorBackupTask(ScyllaOperatorTaskBaseClass):
+    # DC list of datacenter glob patterns, e.g. 'dc1', '!otherdc*' used to specify the DCs
+    # to include or exclude from backup.
+    dc: List[str] = None
```

Comment:
why we need to repeat all those and their docs ? it's not exactly clear to me. this is from the operator docs ? or from `sctool` docs ?

i.e. now we have multiple places to update on each `scylla-manager` new release (and they move around and add new flags, almost at any release)